### DISCUSSION TITLE
Instructions for Mac installation

### DISCUSSION THREAD

**Discussion by absalone** [tefkah/zotero-night#29] [Q&A] [answered]
Hi there,
How do I install this plugin on mac version of Zotero?
I've downloaded the files, but I don't see the instructions to make it work.
Thanks

**Comment by tefkah** [comment, marked as answer]
Ah sorry, I kind of assumed assumed that people using this were familiar with installing Zotero extensions, but it's good to mention it explicitly. 
Here's a video detailing the process

https://user-images.githubusercontent.com/21983833/168032714-6106b138-2725-4091-830b-770dbdff43a4.mov

**Reply by absalone** [reply]
Thanks! It works

**Comment by Sebastian1611** [comment]
What is the latest version for Mac? Kind regards

**Reply by tefkah** [reply]
There is no version difference per operating systems, the instructions  are the same as above

**Reply by Sebastian1611** [reply]
Nevermind, I just saw the sun/ moon symbol on the side to turn it on and off - and I didn't see it before. Thanks a lot for your plugin, I love using it!
